backend/backend/settings/deps/path.py

```diff
@@ -2,7 +2,7 @@
 import sys
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))
 PROJECT_ROOT = os.path.dirname(BASE_DIR)
```

Comment:
Would this be clearer?

```suggestion
from pathlib import Path

# [0] == current dir, [2] == 2 levels up
BASE_DIR = Path(__file__).resolve().parents[2]
PROJECT_ROOT = BASE_DIR.parent
```

L9 could also be `sys.path.append(PROJECT_ROOT / "libs")`

/ ! \ Seems that Django can use pathlike, otherwise indeed this solution would require `str()`

Reply:
It looks to be working and that seems to be [how Django recommends to do it](https://github.com/django/django/blob/107865780aa44914e21d27fdf4ca269bc61c7f01/docs/howto/overriding-templates.txt#L32) (the actual `PROJECT_ROOT` and `BASE_DIR` settings are undocumented, oh well).